packages/@aws-cdk/aws-events-targets/lib/log-group.ts

```diff
@@ -0,0 +1,52 @@
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as logs from '@aws-cdk/aws-logs';
+import * as cdk from '@aws-cdk/core';
+import { LogGroupResourcePolicy } from './log-group-resource-policy';
+/**
+ * Customize the CloudWatch LogGroup Event Target
+ */
+export interface LogGroupProps {
+  /**
+   * The event to send to the CloudWatch LogGroup
+   *
+   * This will be the event logged into the CloudWatch LogGroup
+   *
+   * @default - the entire EventBridge event
+   */
+  readonly event?: events.RuleTargetInput;
+}
+
+/**
+ * Use an AWS CloudWatch LogGroup as an event rule target.
+ */
+export class LogGroup implements events.IRuleTarget {
```

Comment:
I think this should be named CloudWatchLogGroup to align with the naming convention of the other classes the implement `IRuleTarget` (i.e. `CodeBuildProject`, `SnsTopic`, `SqsQueue` ...)